service/src/test/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCacheTest.java

```diff
@@ -119,14 +129,18 @@ public void testGetWithPolicyInvalidation() throws Exception {
         assertEquals(cachedResult.getEffect(), result.getEffect());
 
         Thread.sleep(1);
-        this.cache.resetForPolicySet(ZONE_NAME, "policyOne");
+        this.cache.resetForPolicySet(ZONE_NAME, POLICY_ONE.getName());
         assertNull(this.cache.get(key));
     }
 
     @Test
     public void testGetWithResetForMultiplePolicySets() throws Exception {
 
         PolicyEvaluationRequestV1 request = new PolicyEvaluationRequestV1();
+        this.cache.resetForSubject(ZONE_NAME, AGENT_MULDER);
+        this.cache.resetForResource(ZONE_NAME, XFILES_ID);
+        this.cache.resetForPolicySet(ZONE_NAME, POLICY_ONE.getName());
+        this.cache.resetForPolicySet(ZONE_NAME, POLICY_TWO.getName());
         request.setAction(ACTION_GET);
```

Comment:
Do we really need to call resets for each entity?  Get will return cached result if no timestamps.